README.md

```diff
@@ -5,13 +5,12 @@ Node.js API**  also known as **N-API**, providing C++ object model and exception
 handling semantics with low overhead. It guarantees backward compatibility with
 use with older versions of Node.js that do not have N-API built-in.
 
-Node.js API guarentees the **API** and **ABI** compatibility across different
-version of Node.js. So if you switch to a
-different version of Node.js you must not reinstall and maybe recompile the
-Addon.
+N-API guarantees the **API** and **ABI** compatibility across different
+versions of Node.js. So if you switched to a different version of Node.js,
+you would not need to reinstall or recompile the Addon.
 
 N-API is an API for building native Addons. It is independent from the underlying
-JavaScript runtime (ex V8) and is maintained as part of Node.js itself. This API
+JavaScript runtime (e.g. v8) and is maintained as part of Node.js itself. This API
```

Comment:
I think this should be V8 as I believe I saw a change somewhere else to make updates to have it be consistently V8 as opposed to v8.